src/main/scala/com/wavesplatform/state2/StateStorage.scala

```diff
@@ -1,120 +1,92 @@
 package com.wavesplatform.state2
 
-import java.io.File
-
 import com.google.common.primitives.Ints
-import com.wavesplatform.utils._
-import org.h2.mvstore.MVMap
 import scorex.account.Address
-import scorex.utils.LogMVMapBuilder
+import org.iq80.leveldb.DB
 
-import scala.util.Try
+import vsys.db.SubStorage
+import vsys.db.StateMap
 
-class StateStorage private(file: Option[File]) extends AutoCloseable {
+class StateStorage private(db: DB) {
 
   import StateStorage._
 
-  private val db = createMVStore(file)
-
-  private val variables: MVMap[String, Int] = db.openMap("variables")
+  private val variables: StateMap[String, Int] = new StateMap(db, "variables")
 
-  private def setPersistedVersion(version: Int) = variables.put(stateVersion, version)
+  // private def setPersistedVersion(version: Int) = variables.put(stateVersion, version)
```

Comment:
is this not needed any more?

Reply:
After discussion with @ncying, The ```PersistedVersion``` is saved in new commits